ProjectSettings/ProjectSettings.asset

```diff
@@ -791,7 +791,7 @@ PlayerSettings:
   webGLTemplate: PROJECT:Netherlands3D
   webGLAnalyzeBuildSize: 0
   webGLUseEmbeddedResources: 0
-  webGLCompressionFormat: 1
```

Comment:
Zijn deze changes bewust? Dat de compression en selectedPlatform gewijzigd zijn?

Reply:
ai dat is niet de bedoeling. dit was zodat ik op azure kon deployen die .br ondersteuning heeft ipv. gzip. zal m' terugzetten